### PR TITLE
Zhuravleva settings

### PR DESCRIPTION
**Pull request by Zhur-a-vleva**

### CODE REVIEW

File: reusable_learning_app/lib/screens/Content/settings_body.dart [hunk 1]
```diff
@@ -13,18 +14,261 @@ class SettingsBody extends StatefulWidget implements NavItem   {
...
+                            value: _isAudioOn,
+                            activeColor: Colors.deepPurple,
+                            onChanged: (value) {
+                              _changeAudioMode(value);
+                            })
+                      ],
+                    ),
+                  ))) // Audio
+          ,
+          Padding(
```
Comment (Orillio) [inline reusable_learning_app/lib/screens/Content/settings_body.dart:224]: Make "notification" and "get support" options clickable not only with arrow icon.

image

File: reusable_learning_app/lib/screens/Content/settings_body.dart [hunk 2]
```diff
@@ -13,18 +14,261 @@ class SettingsBody extends StatefulWidget implements NavItem   {
 }
 
 class SettingsBodyState extends State<SettingsBody> {
```
Comment (Orillio) [inline reusable_learning_app/lib/screens/Content/settings_body.dart:16]: Please, make sure you follow the overall design of the app. Shadows are too visible and blurry. Make them similar as we have in the prototype.

<div style="display:flex; flex-direction: row; flex-">
  image
  image
</div>

**Review by Orillio — CHANGES_REQUESTED** [review]
Please provide changes described in review

File: reusable_learning_app/lib/screens/Content/settings_body.dart (hunk 2 above)
Comment (Zhur-a-vleva) [reply reusable_learning_app/lib/screens/Content/settings_body.dart:16]: Done. Changes commited

File: reusable_learning_app/lib/screens/Content/settings_body.dart (hunk 1 above)
Comment (Zhur-a-vleva) [reply reusable_learning_app/lib/screens/Content/settings_body.dart:224]: Done

**Review by Orillio — APPROVED** [review]

**Pull request merged**